content/en/tracing/guide/send_traces_to_agent_by_api.md

```diff
@@ -56,7 +58,7 @@ and each span is a dictionary with a `trace_id`, `span_id`, `resource` and so on
 | `service`    | string  | The service you are tracing. The service name must not be longer than 100 characters. |
 | `span_id`    | int64   | The span integer (64-bit unsigned) ID. |
 | `start`      | int64   | The start time of the request in nanoseconds from the UNIX epoch. |
-| `trace_id`   | int64   | The unique integer (64-bit unsigned) ID of the trace containing this span. |
+| `trace_id`   | int64 or int128   | The unique integer (64-bit unsigned or 128-bit unsigned) ID of the trace containing this span. |
```

Comment:
It depends a bit on what `trace_id` means here.

If we are talking about the logical `trace_id` field then this is fine.

However, our msgpack and protobuf representations of trace data contain a uint64 trace_id field, which contains the lower-order 64-bits of the trace id - if that's what's being referred to, customers will find this confusing. They can use the lower-order 64 bits of the trace id to fill in this field and we will match it up to the 128-bit trace id in the backend.

Reply:
Thanks for taking a look. In my opinion, we should focus on the logical `trace_id` field. As you mentioned before as well, we plan to change this representation eventually so it'd be best not to expose these details.